frontend/src/resources/utils/get-cluster.ts

```diff
@@ -794,6 +802,8 @@ export function getClusterStatus(
                     provisionFailedCondition?.reason === 'InvalidInstallConfig'
                 ) {
                     cdStatus = ClusterStatus.provisionfailed
+                } else if (provisionCurrentlyFailing) {
+                    cdStatus = ClusterStatus.provisionfailed
                 } else {
                     cdStatus = ClusterStatus.creating
```

Comment:
Can you provide the ClusterDeployment YAML from the environment you were able to access?

Reply:
This is a tricky one. I wasn't able to figure out why we check the message for `currentProvisionRef` either. Not sure if @showeimer might remember... this was from December of last year: https://github.com/open-cluster-management/console/commit/a8019f580c05f0cd4f6fd28581c345b7b1dec934

I'm not sure if the condition being the last one is significant.

I'm wondering if we should actually remove the check for `currentProvisionRef` and favour "failed" over "creating", since I would think it's not terribly common that we fail and then succeed on another attempt. But on the other hand, the error here is about quota, so that is something that could vary over time.

I do not see any recent jobs for the ClusterDeployment, so it looks like it has stopped trying, yet the `ProvisionStopped` condition is false. Actually, the jobs are about 6 hours apart... not sure if that is usual. I guess if it is still trying, we should see another one soon. I have asked in #forum-hive about whether it ever stops trying.

@jamestalton Thoughts on this one?